src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig

```diff
@@ -234,3 +235,11 @@
     </div>
     {%- endif %}
 {%- endblock form_errors %}
+
+{# Help #}
+
+{% block form_help -%}
+    {% if form.vars.help is not empty %}
+        <small class="form-text text-muted">{{ form.vars.help|raw }}</small>
```

Comment:
displaying it raw is a bad idea IMO. It opens the door to XSS.

Reply:
Mmm, is it really a problem ? Because it's set in the FormType by the dev, not by a user. Then, if we want use html in the help text without raw, we can't. I don't really know

Reply:
Well, requiring to perform HTML escaping when setting the option is insecure IMO. It requires all devs to be aware that this options requires escaping when setting it. And it might totally be set based on some user input (coming from a previous configuration elsewhere) in some cases.

Btw, I would add translation support for the help text instead of requiring to translate it in the form type.

Reply:
Ok, I remove it from Symfony, if someone want to change it, just to extend the Template and add the raw.

By translation you mean use `{% trans %}` in Twig ? (I've never use it for the moment)

Reply:
yes, as done for labels or other things